ocs_ci/utility/uninstall_openshift-logging.py

```diff
@@ -0,0 +1,53 @@
+"""
+Function to teardown the openshift-logging
+"""
+import logging
+
+from ocs_ci.ocs import constants
+from ocs_ci.ocs import ocp
+
+logger = logging.getLogger(__name__)
+
+
+def uninstall_cluster_logging():
+
+    """
+    Function to uninstall cluster-logging from the cluster
+    1. Deletes the project "openshift-logging"
+    & "openshift-operators-redhat"
+    2. Deletes the CSC resource "cluster-logging-operator"
+    & "elasticsearch"
+    """
+    obj1 = ocp.OCP(
```

Comment:
Can we use more descriptive var names instead of `obj1` and `obj2`? These are too vague.